ffw/BasicCommunicationRPC.js

```diff
@@ -500,8 +500,9 @@ FFW.BasicCommunication = FFW.RPCObserver
           }
         }
         if (notification.method == this.onAppRegisteredNotification) {
+          let appModel = Object.assign(notification.params.application, {"priority": notification.params.priority}); 
```

Comment:
```suggestion
          let appModel = Object.assign(notification.params.application, {"priority": notification.params.priority ? notification.params.priority : 'NONE'}); 
```

you may want to make this multiple lines :)